packages/terra-dialog/src/Dialog.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Dialog.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+ /*
+ * Content to be displayed as the name
+ */
+  name: PropTypes.string,
+};
+
+const defaultProps = {
+  name: 'default',
+};
+
+const Dialog = ({ name, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
+  const DialogClassNames = cx([
+    'dialog',
+    attributes.className,
+  ]);
+
+  return (<div {...attributes} className={DialogClassNames} />);
```

Comment:
Add `{name}` as text. This [PR](https://github.com/cerner/generator-terra-module/pull/81) fixes this but it hasn't been merged.

Reply:
https://github.com/cerner/terra-core/pull/1158/commits/f7c833de381df9ada04ac26c89e11b89c660355e